docker-compose.yaml

```
@@ -1,14 +1,14 @@
 networks:
-    interns2025c-dev:
+    LapGo-dev:
         driver: bridge
     traefik-proxy-blumilk-local-environment:
       external: true
 
 volumes:
-    interns2025c-postgres-data:
-        name: interns2025c-postgres-data
-    interns2025c-redis-data:
-        name: interns2025c-redis-data
+    LapGo-postgres-data:
+        name: LapGo-postgres-data
+    LapGo-redis-data:
+        name: LapGo-redis-data
```

```suggestion
    lapgo-postgres-data:
        name: lapgo-postgres-data
    lapgo-redis-data:
        name: lapgo-redis-data
```

Lets stay with lowercased networks, volumes names etc.